### ISSUE TITLE
Basalt log spam (1.20.4 fabric)

### ISSUE THREAD

**Issue by Nyarumi** [yungnickyoung/YUNGs-Better-Dungeons#62]
My logs are being spammed with this message by Distant Horizons about broken basalt columns.

`[22:19:04] [DH-World Gen Thread[0]/WARN]: Better Dungeons: Detected a mod with a broken basalt columns configuredfeature that is trying to place blocks outside the 3x3 safe chunk area for features. Find the broken mod and report to them to fix the placement of their basalt columns feature.`

This looks to be the [exact same issue](https://github.com/YUNG-GANG/YUNGs-Better-Fortresses/issues/13) as the one with Better Fortresses that was fixed recently. I’m not sure why it’s triggering with Better Dungeons as I doubt it has this feature, but it doesn’t seem to care.

**Comment by mattazz** [comment]
Same is happening with me, it's infrequent but it's the only thing that's causing my game to crash

**Comment by abby-cadabra** [comment]
I'm having the same on 1.20.1 Forge. Caused my nether to corrupt and I had to regenerate it. Also caused the latest.log to bloat up to 3GB. 

[13:35:24] [DH-World Gen Thread[3]/WARN]: Repurposed Structures: Detected a mod with a broken basalt columns configuredfeature that is trying to place blocks outside the 3x3 safe chunk area for features. Find the broken mod and report to them to fix the placement of their basalt columns feature.
[13:35:24] [DH-World Gen Thread[3]/WARN]: Cataclysm: Detected a mod with a broken basalt columns configuredfeature that is trying to place blocks outside the 3x3 safe chunk area for features. Find the broken mod and report to them to fix the placement of their basalt columns feature.
[13:35:24] [DH-World Gen Thread[3]/WARN]: Better Dungeons: Detected a mod with a broken basalt columns configuredfeature that is trying to place blocks outside the 3x3 safe chunk area for features. Find the broken mod and report to them to fix the placement of their basalt columns feature.
[13:35:24] [DH-World Gen Thread[3]/WARN]: Repurposed Structures: Detected a mod with a broken basalt columns configuredfeature that is trying to place blocks outside the 3x3 safe chunk area for features. Find the broken mod and report to them to fix the placement of their basalt columns feature.
[13:35:24] [DH-World Gen Thread[3]/WARN]: Cataclysm: Detected a mod with a broken basalt columns configuredfeature that is trying to place blocks outside the 3x3 safe chunk area for features. Find the broken mod and report to them to fix the placement of their basalt columns feature.
[13:35:24] [DH-World Gen Thread[3]/WARN]: Better Dungeons: Detected a mod with a broken basalt columns configuredfeature that is trying to place blocks outside the 3x3 safe chunk area for features. Find the broken mod and report to them to fix the placement of their basalt columns feature.
[13:35:24] [DH-World Gen Thread[3]/WARN]: Repurposed Structures: Detected a mod with a broken basalt columns configuredfeature that is trying to place blocks outside the 3x3 safe chunk area for features. Find the broken mod and report to them to fix the placement of their basalt columns feature.
[13:35:24] [DH-World Gen Thread[3]/WARN]: Cataclysm: Detected a mod with a broken basalt columns configuredfeature that is trying to place blocks outside the 3x3 safe chunk area for features. Find the broken mod and report to them to fix the placement of their basalt columns feature.
[13:35:24] [DH-World Gen Thread[3]/WARN]: Better Dungeons: Detected a mod with a broken basalt columns configuredfeature that is trying to place blocks outside the 3x3 safe chunk area for features. Find the broken mod and report to them to fix the placement of their basalt columns feature.
[13:35:24] [DH-World Gen Thread[3]/WARN]: Repurposed Structures: Detected a mod with a broken basalt columns configuredfeature that is trying to place blocks outside the 3x3 safe chunk area for features. Find the broken mod and report to them to fix the placement of their basalt columns feature.

**Comment by yungnickyoung** [comment]
Sorry about this. v4.0.4 for 1.20.1 and v4.4.2 for 1.20.4 remove this log spam. Both are available on modrinth now (waiting on CF approval). That should solve this issue, please feel free to make a new issue report if the problem persists!
